test/unit/plugin_test.rb

```diff
@@ -365,7 +365,12 @@ def test_register_status_extension
         api: {
           version: Apipie.configuration.default_version,
         },
-        plugins: [Foreman::Plugin.find(:foo)],
+        plugins: [
```

Comment:
To be more accurate, I think it should be something like:
```ruby
...
plugin = Foreman::Plugin.find(:foo)
...
plugins: [ { name: plugin.id, version: plugin.version } ]
...
```

Reply:
I think it's a good practice to have exact values in your expected results. Otherwise you may miss a situation where a property may be nil for example. Essentially there is no assertion then.

Reply:
I had the dynamic version before, and then had the same thought as Ewoud: lets test explicit values.